releasenotes/notes/mps_improve_swaps-e7a142ca3b8644cd.yaml

```diff
@@ -0,0 +1,8 @@
+other:
+  - |
+    Improved performance ofalgorithm for swapping of qubits in MPS 2-qubit gates.
```

Comment:
"of algorithm"

Reply:
@mtreinish , @hhorii , @chriseclectic - this PR was approved by @yaelbh . Can one of you please merge?